optuna/integration/chainer.py

```diff
@@ -28,15 +28,12 @@ class ChainerPruningExtension(Extension):
 
     Example:
```

Comment:
I think this `Example:` is mostly just noisy and that it can be removed. What do you think?

Reply:
I removed `Example:` from docstring, applied same thing to other files.